src/ResultWriter/ReceiverWriter.cpp

```diff
@@ -47,6 +47,44 @@
 #include <Parallel/MPI.h>
 #include <Modules/Modules.h>
 
+#include <sstream>
+#include <string>
+#include <fstream>
+#include <regex>
+
+Eigen::Vector3d seissol::writer::parseReceiverLine(const std::string& line) {
+  std::regex rgx("\\s+");
+  std::sregex_token_iterator iter(line.begin(),
+                                  line.end(),
+                                  rgx,
+                                  -1);
+  std::sregex_token_iterator end;
+  std::array<double, 3> coordinates{};
+  unsigned numberOfCoordinates = 0;
+  for (; iter != end; ++iter, ++numberOfCoordinates) {
+    if (numberOfCoordinates > coordinates.size()) {
+      break;
+    }
+    coordinates[numberOfCoordinates] = std::stod(*iter);
+  }
+  if (numberOfCoordinates != coordinates.size()) {
+    throw std::runtime_error("Incorrect number of coordinates in line " + line + ".");
+  }
+  return {coordinates[0], coordinates[1], coordinates[2]};
+}
+
+std::vector<Eigen::Vector3d> seissol::writer::parseReceiverFile(const std::string& receiverFileName) {
+  std::vector<Eigen::Vector3d> points{};
+
+  std::ifstream file{receiverFileName};
+  std::string line{};
+  while (std::getline(file, line)) {
+    bool onlyWhiteSpace = std::all_of(line.begin(), line.end(), isspace);
```

Comment:
Can you make it more verbose: `std::isspace`?

Reply:
Unfortunately, this is just more ugly:
https://stackoverflow.com/a/46208892
std::isspace is overloaded, isspace not. Not sure why...